cylc/flow/parsec/validate.py

```diff
@@ -1007,22 +1007,30 @@ def coerce_parameter_list(cls, value, keys):
             >>> CylcConfigValidator.coerce_parameter_list('a, b, c', None)
             ['a', 'b', 'c']
 
+            >>> CylcConfigValidator.coerce_parameter_list('084_132', None)
+            ['084_132']
+
+            >>> CylcConfigValidator.coerce_parameter_list('72, a', None)
+            ['72', 'a']
```

Comment:
I think this test probably passed before, maybe worth trying a zero prefix?

```suggestion
            >>> CylcConfigValidator.coerce_parameter_list('072, a', None)
            ['072', 'a']
```

Reply:
Oliver's comment hasn't been resolved

Reply:
I've accepted it, because it does not harm, but Oliver's original comment missed the point that this test fixes a coverage hole rather than being related to the matter in hand.